cfme/containers/provider/openshift.py

```diff
@@ -42,30 +42,7 @@ def get_ca_cert(connection_info):
             return str("".join(stdout.readlines()))
 
 
-class HawkularEndpoint(DefaultEndpoint):
-    """Represents Hawkular Endpoint"""
-    name = 'hawkular'
-
-    @property
-    def view_value_mapping(self):
-        out = {
-            'hostname': self.hostname,
-            'api_port': self.api_port,
-            'sec_protocol': self.sec_protocol
-        }
-
-        if out['sec_protocol'] and self.sec_protocol.lower() == 'ssl trusting custom ca':
-            out['trusted_ca_certificates'] = OpenshiftDefaultEndpoint.get_ca_cert(
-                {"username": self.ssh_creds.principal,
-                 "password": self.ssh_creds.secret,
-                 "hostname": self.master_hostname})
-
-        return out
-
-
-class AlertsEndpoint(DefaultEndpoint):
-    """Represents Alerts Endpoint"""
-    name = 'alerts'
+class ServiceBasedEndpoint(DefaultEndpoint):
```

Comment:
I need to do some local testing to confirm, but this does not look like it will support configuration of the metrics endpoints (hawkular tab) in 58z, since the view_value_mapping will return nothing.

How has your local testing of 58z and ocp-v1/v2 worked with their hawkular endpoints?  Too bad PRT is in a deadlock and we can't merge the yamls PR first because it will break collection :/

Reply:
```view_value_mapping``` returns a ```dict``` object with all the endpoint info.

you can see this at https://github.com/shalomnaim1/integration_tests/blob/59a1ae7fbd83251214e2c54bb4e5add9043766ac/cfme/containers/provider/openshift.py#L68

Reply:
ocp-v2 is not supported in 5.8 - I think (documentation for 4.5 kinda stinks). But ocp-v1 should be checked.

Reply:
@jawatts I was able to add ```ocp-v1``` to 5.8 appliances. about ```ocp-v2``` I didn't check that.